dev/prod_builders.json

```diff
@@ -1152,6 +1158,12 @@
       "task_name": "mac_ios_platform_channel_sample_test_swift",
       "flaky": false
     },
+    {
+      "name": "Mac_ios platform channels benchmarks",
```

Comment:
oh, nit - if this can run on Linux instead of mac it would be better. We have many more Linux builders than mac.

Reply:
Makes sense but I'm not comfortable making that change because I don't have a linux setup where I can run it locally to make sure it works and microbenchmarks runs on mac which is a test which is somewhat similar.

Reply:
Sorry, I mean the Android one - not the Linux one. There shouldn't really be any significant difference, sinc ethis test is really running on the device right?

Reply:
done